api4/integration_action_test.go

```diff
@@ -237,7 +237,7 @@ func TestSubmitDialog(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		var request model.SubmitDialogRequest
 		err := json.NewDecoder(r.Body).Decode(&request)
-		require.Nil(t, err)
+		require.NoError(t, err)
 		assert.NotNil(t, request)
```

Comment:
Actually, the pointer is always to be not-nil because is a pointer to a variable that always exists. I think we can remove this line because later is checking the values of request (so... in case of no error, it should have the proper data).